src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -223,20 +182,30 @@ public boolean runUntilEnd() throws IOException, InterruptedException {
 
     @Override
     List<String> getTargetSpecificAOTCompileFlags() throws IOException {
-        return Arrays.asList("-H:CompilerBackend=" + Constants.BACKEND_LLVM,
+        ArrayList<String> flags = new ArrayList<String>(Arrays.asList(
                 "-H:-SpawnIsolates",
                 "-Dsvm.targetArch=" + projectConfiguration.getTargetTriplet().getArch(),
                 "-H:+UseOnlyWritableBootImageHeap",
                 "-H:+UseCAPCache",
                 "-H:CAPCacheDir=" + getCapCacheDir().toAbsolutePath().toString(),
-                "-H:CustomLD=" + ldlld.toAbsolutePath().toString());
+                "-H:+UseBionicC",
```

Comment:
This fails on GraalVM 20.1
Can we make this optional until there is a GraalVM release that has this flag?

Reply:
Maybe we should add GraalVM version detection for this kind of things?